core/src/lib/output_formatter.cc

```diff
@@ -878,7 +1056,7 @@ void OutputFormatter::JsonFinalizeResult(bool result)
     /*
      * send json string, on failure, send json error message
      */
-    if (!send_func(send_ctx, string)) {
+    if (!send_func(send_ctx, "%s", string)) {
```

Comment:
This will never be false.

Reply:
It can fail, if there is a network problem.